src/style.css

```diff
@@ -0,0 +1,36 @@
+
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: center;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(images/star.svg);
+}
```

Comment:
You could use this image for every star. Than you just rewrite it with code below.